### ISSUE TITLE
AR "Split Aggregate by Entities" fails (V6.5, Win)

### ISSUE THREAD

**Issue by socadk** [ContextMapper/context-mapper-dsl#292]
When I apply the AR to this CML

```
BoundedContext SampleContext {
	domainVisionStatement "This BC models..."
	type FEATURE
	knowledgeLevel CONCRETE
	Aggregate SampleAggregate {
		DomainEvent SomethingGotCreated
		Entity SampleEntity1 {
			def String getStatus;
		}
		Entity SampleEntity2 {
			def String doSomething;
		}
	}
}
```

I get

![image](https://user-images.githubusercontent.com/5002426/119870672-e878f800-bf21-11eb-9c63-21c0b49fc5d3.png)

**Comment by stefan-ka** [comment]
This one should be fixed with v6.5.2. Can you check that, @socadk? Thanks!

**Comment by socadk** [comment]
Works, thanks!
